barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.hpp

```diff
@@ -269,14 +267,13 @@ class ClientIVC {
   public:
     size_t num_circuits_accumulated = 0; // number of circuits accumulated so far
 
-    ProverFoldOutput fold_output; // prover accumulator and fold proof
```

Comment:
we only track the `accumulator` since the proof is stored internal to the verification queue